test/recipeModuleManager.test.js

```diff
@@ -45,4 +57,674 @@ describe(`recipeModuleManagerTest: ${printPath("[test/recipeModuleManager.test.j
         await killAllST();
         await cleanST();
     });
+
+    // * TODO: Check that querier has been inited when we call supertokens.init
+    it("test that querier has been initiated when we call supertokens.init", async function () {
+        await startST();
+
+        try {
+            await Querier.getInstanceOrThrowError();
+            assert(false);
+        } catch (err) {}
+
+        ST.init({
+            supertokens: {
+                connectionURI: "http://localhost:8080",
+            },
+            appInfo: {
+                apiDomain: "api.supertokens.io",
+                appName: "SuperTokens",
+                websiteDomain: "supertokens.io",
+            },
+            recipeList: [Session.init()],
+        });
+
+        await Querier.getInstanceOrThrowError();
+    });
+
+    // TODO: Check that modules have been inited when we call supertokens.init
+    it("test that modules have been initiated when we call supertokens.init", async function () {
+        await startST();
+
+        try {
+            await SessionRecipe.getInstanceOrThrowError();
+            await EmailPasswordRecipe.getInstanceOrThrowError();
+            assert(false);
+        } catch (err) {}
```

Comment:
- Again, no checking of error type
- `await EmailPasswordRecipe.getInstanceOrThrowError();` never actually runs... cause `await SessionRecipe.getInstanceOrThrowError();` throws an error.. So you should have these two in different try catch